src/components/VSelect/VSelect.js

```diff
@@ -45,7 +46,8 @@ export default {
     VListTileContent,
     VListTileTitle,
     VMenu,
-    VBtn
+    VBtn,
+    Dependent
```

Comment:
Update to match this spec https://docs.google.com/document/d/1xWWnAw2zYce3sVvdiRC0OuIhNx1PCt-oCNzTXeREFaw/edit?usp=sharing